src/components/Popover/Popover.module.css

```
@@ -1,9 +1,119 @@
+  --padding: 10px;
+  --border-width: 2px;
+  --margin: 10px;
+
+  right: calc(anchor(left));
+  align-self: anchor-center;
+  position-try-fallbacks:
+    flip-block,
+    flip-inline,
+    flip-block flip-inline;
+
+  margin: var(--margin) 15px var(--margin) var(--margin);
```

Stylelint (17.4.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)